config/internal/webhook/service.yaml.tmpl

```diff
@@ -0,0 +1,19 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{.WebhookName}}
+  namespace: {{.DSPONamespace}}
+  annotations:
+    service.beta.openshift.io/serving-cert-secret-name: ds-pipelines-webhook-tls-{{.WebhookName}}
```

Comment:
```suggestion
    service.beta.openshift.io/serving-cert-secret-name: {{.WebhookName}}-webhook-tls
```

Reply:
@mprahl How about `{{.WebhookName}}-tls ` ? since WebhookName is "kubernetes-webhook"

Reply:
Yes, good idea.